src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -579,20 +587,25 @@ public void TotalProcessorTime_GetNotStarted_ThrowsInvalidOperationException()
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/corefx/issues/22266", TargetFrameworkMonikers.UapNotUapAot)]
         public void TestProcessStartTime()
         {
-            TimeSpan allowedWindow = TimeSpan.FromSeconds(3);
-            DateTime testStartTime = DateTime.UtcNow;
-            using (var remote = RemoteInvoke(() => { Console.Write(Process.GetCurrentProcess().StartTime.ToUniversalTime()); return SuccessExitCode; },
-                new RemoteInvokeOptions { StartInfo = new ProcessStartInfo { RedirectStandardOutput = true } }))
-            {
-                Assert.Equal(remote.Process.StartTime, remote.Process.StartTime);
+            TimeSpan allowedWindow = TimeSpan.FromSeconds(1);
```

Comment:
@stephentoub I'm not seeing any reason why this should ever be outside of the range at all but this seems to happen on some OSes - looking at the failures with exact range the times seem to be the same (ToString()) which suggests difference in milliseconds (possibly time coming from different sources and have different precision - I haven't investigated) - I think 1s should be enough to cover all precision differences but I did not investigate why 3 was chosen here before. Do you perhaps have some background on that? I've seen you have edited this test previously